README.md

```diff
@@ -19,6 +19,7 @@ Read More: https://www.civo.com/learn/managing-external-load-balancers-on-civo
 | Annotation | Description | Example Values |
 |------------|-------------|----------------|
 | kubernetes.civo.com/firewall-id | If provided, an existing Firewall will be used. | 03093EF6-31E6-48B1-AB1D-152AC3A8C90A |
+| kubernetes.civo.com/max-concurrent-requests | If provided, the user can specify the max number of concurrent requests a loadbalancer could afford | 10000, 20000, 30000, 40000, 50000 |
```

Comment:
Just add a comment around downtime once user sets/changes this value

Reply:
Added.